CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 cmake_minimum_required(VERSION 3.20)
```

Comment:
Revert this file, we should require C++23, we have not versioning yet and description is point of discussions